src/System.Private.Xml/src/System.Private.Xml.builds

```diff
@@ -3,12 +3,12 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Private.Xml.csproj" />
-    <Project Include="System.Private.Xml.csproj">
+    <!--<Project Include="System.Private.Xml.csproj">
```

Comment:
I believe that is true as well, but it should be verified. If it is true then we should delete the build configurations instead of commenting them out.